### DISCUSSION TITLE
CKB Dev Log 2024-04-18

### DISCUSSION THREAD

**Discussion by linnnsss** [nervosnetwork/ckb#4421] [Dev Log]
## Updates

***Releases***

- **CKB v0.115.0**: Now available for download. [Release Notes](https://github.com/nervosnetwork/ckb/releases/tag/v0.115.0)

***Features***

- **Omnilock** now supports Solana Phantom Wallet https://github.com/cryptape/omnilock/pull/6
- **CKB Payment Channel Network (PCN)**
    - Channel design and implementation https://github.com/contrun/ckb-pcn-node/tree/channel
    - Implementing PCN Hashed Time Lock Contracts (HTLC) Cross-Chain Pitch
    - Invoice protocol design
    - Finalizing HTLC lock https://github.com/quake/ckb-pcn-scripts
- **CKB VM**
    - Syscall spawn PR ready for review https://github.com/nervosnetwork/ckb/pull/4380
        - Tests added
            - DAG tests: https://github.com/libraries/ckb/commit/7f5697a926fed9142a176f6fc6b6d924db85f8b0
            - Fuzzing tests: https://github.com/libraries/ckb/pull/17
    
***Improvements***

- **CKB CoBuild**
    - ckb-rust-sdk now supports cobuild
    - Reorganize the previous internal cobuild essays into public available ones, including:
        - [*Approved Action: Exploring Exotic Cobuild Actions*](https://talk.nervos.org/t/approved-action-exploring-exotic-cobuild-actions/7998/1)
        - *Cobuild behind the scenes*
        
***Fixes***

- **Asynchronous Block Handling**
    - Integration testing: https://github.com/nervosnetwork/ckb/pull/4365/commits/cba0b481db0ebd7c4be03a5d81ba99af00d93187
    - Code review and bug fixes

## In Pipeline…

- **CKB Payment Channel Network (PCN)**
    - How to support sUDT/xUDT
    - Design Commitment TX lock
    - The p2p funding and commitment tx interface
- **CKB Transaction CoBuild Protocol**
    - Build a cobuild OTX demo (AMM & Orderbook DEX contract, OTX processor)
- **CKB Information Collection Optimization**
    - Node monitoring dashboard
- **New CKB release v0.116.0**
- **Syscall spawn RFC and documentation**

**Comment by joshyates1980** [comment]
We appreciate the good, detailed release notes after every update. Thank you.
